roles/edpm_neutron_metadata/molecule/default/prepare.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+- name: Prepare test_deps
+  hosts: all
+  roles:
+    - role: ../../../../molecule/common/test_deps
+      test_deps_setup_edpm: true
+      test_deps_setup_stream: true
+      test_deps_extra_packages:
+        - openvswitch
+        - iproute
+        - podman
+- name: Prepare
+  hosts: all
+  gather_facts: true
+  tasks:
+    - include_role:
+        name: osp.edpm.env_data
+
+    # The openvswitch kernel module needs to be loaded on the host
+    - name: install and modprobe openvswitch
+      shell: |
+        sudo dnf -y install openvswitch
+        sudo modprobe openvswitch
+      delegate_to: localhost
+      run_once: true
```

Comment:
It would be better to implement a block with 2 different tasks, `ansible.builtin.dnf` and `community.general.modprobe`. Avoid using `command` and `shell` (this one is *slow* and should be avoided anyway if shell functions like piping or redirections aren't involved) modules when they aren't strictly needed.

Reply:
Ohh, ok, I copied it from what we have in ovn and other edpm_neutron_X roles

Reply:
I'm getting this error, so I'm reverting back to the command/shell option:
`        "fatal: [instance -> localhost]: FAILED! => {\"changed\": false, \"msg\": \"Could not import the dnf python module using /opt/hostedtoolcache/Python/3.9.18/x64/bin/python (3.9.18 (main, Aug 28 2023, 08:38:32) [GCC 11.4.0]). Please install `python3-dnf` or `python2-dnf` package or ensure you have specified the correct ansible_python_interpreter. (attempted ['/usr/libexec/platform-python', '/usr/bin/python3', '/usr/bin/python2', '/usr/bin/python'])\", \"results\": []}",`

Reply:
I was trying to do similar change in edpm_neutron_dhcp role where we are doing the same but unfortunatelly it's not that easy as AFAIK molecule don't allows to ask for sudo password when "become: true" is used so this is failing when doing it in the "proper" way.
This hacky usage of shell module however works fine as locally it will simply stop and ask for sudo password :)

Reply:
In molecule testing we shouldn't enforce things since it's not everything is working as expected, as you already pointed out. We'll do some test with ansible-lint in the future, but, for now, we can continue using shell module there.